recipe/conda_build_config.yaml

```diff
@@ -44,10 +47,13 @@ _libgcc_mutex:
 use_system_gcc:
   - True         # [not (linux64 or aarch64 or ppc64le or s390x)]
   - False        # [linux64 or aarch64 or ppc64le or s390x]
+channel_sources:
+  - conda-forge/label/sysroot-with-crypt,conda-forge
```

Comment:
For my own knowledge, why do we need the sysroot with crypt here?

Reply:
`gcc<14`'s libsanitizer compiles against crypt.h and links to libcrypt.a. It's hard to get this package to use our `libxcrypt` package and also creates bootstrapping problems.